grass7/vector/v.class.mlR/v.class.mlR.html

```diff
@@ -1,7 +1,7 @@
 <h2>DESCRIPTION</h2>
 
 <p>
-<em>v.class.mlR</em> is a wrapper script that uses the R caret package 
+<em>v.class.mlR</em> is a wrapper module that uses the R caret package 
 for machine learning in R to classify features using training features 
```

Comment:
To classify objects maybe better? To avoid using features as geometries and variables in the same sentence, which may be confusing